test/jdk/com/sun/net/httpserver/simpleserver/jwebserver/MaxRequestTimeTest.java

```diff
@@ -23,7 +23,7 @@
 
 /*
  * @test
- * @bug 8278398
+ * @bug 8278398 8280161
```

Comment:
No need to add 8280161 since 8280161 is marked as noreg-self. In principle we only list bugs whose fix contain code changes that can be verified by the test.

Reply:
Makes sense, thanks for picking up on that.